plugin/pkg/scheduler/metrics/metrics.go

```diff
@@ -44,6 +42,20 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(1000, 2, 15),
 		},
 	)
+	SchedulingAlgorithmPredicateEvaluationDuration = prometheus.NewGauge(
+		prometheus.GaugeOpts{
```

Comment:
So, we found it easier to represent evaluation time using gauge as we wanted to get accelerometer kind of visualization. The output using gauge would look like this over time, 

![priorityevaluation](https://cloud.githubusercontent.com/assets/11657829/26447545/f68cc152-4116-11e7-92d6-1f075367866f.png)